themes/admin/Sea_Green/config.xml

```diff
@@ -19,6 +19,10 @@
         <lastUpdate>2017-27-11 17:00:00</lastUpdate>
     </metadata>
 
+    <compatibility>
+        <version>6.0</version>
```

Comment:
You don't need to add this to survey themes too?

Reply:
yeah it was in the commit that is outside of this pull request as mentioned in the ticket